### PR TITLE
adding gopigo3 robot

### PR DESCRIPTION
**Pull request by CleoQc**
The GoPiGo now supports ROS, thanks to Bernardo Japon who wrote this book.
https://medium.com/@brjapon/hands-on-ros-with-gopigo3-robot-e8c20c02732f

### CODE REVIEW

File: _posts/2020-02-28-gopigo3.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- 'teaching robot'
+- 'wheeled'
+- 'wifi'
+- 'wireless'
+twitter_text: Checkout the GoPiGo3
+website: http://dexterindustries.com/gopigo
+wiki_homepage: 
+---
+
+TODO Fill in the long description/article here.
```
Comment (tfoote) [inline _posts/2020-02-28-gopigo3.md:31]: Please fill in content here. Otherwise it will just be a blank page.

Comment (CleoQc) [reply _posts/2020-02-28-gopigo3.md:31]: Thanks @tfoote , the Wiki page is now created (Covid threw a wrench in our planning)

**Comment by CleoQc** [comment]
Wiki page added. Thanks!

File: _posts/2020-02-28-gopigo3.md (hunk 1 above)
Comment (tfoote) [reply _posts/2020-02-28-gopigo3.md:31]: This TODO is still pending.  Right now the page for you robot will render with the content 

"TODO Fill in the long description/article here."

**Comment by tfoote** [comment]
Thanks for filling in the wiki page, but the actual content on this site needs to be updated. 

Yeah, things are pretty hectic right now.

**Comment by tfoote** [comment]
Closing due to no response.

**Comment by CleoQc** [comment]
Sorry, COVID has thrown a wrench in the project. We'll be back to it shortly.

**Pull request closed without merging**